src/test/java/plugins/LdapPluginTest.java

```diff
@@ -70,6 +70,34 @@ private LdapDetails createDefaultsWithoutManagerCred(LdapContainer ldapContainer
     }
 
     @Test
+    @WithPlugins("ldap@1.15")
```

Comment:
Well, I do. Red Hat maintains an internal downstream update centers with verified plugins and this is an essential feature for testing it. I presume CB have such internal UCs as well. This will also kick in when not the latest LTS is tested and it reached one of the upstream update center where the latest plugins might be missing as they require newer Jenkins version.

Good point on this still does not work.